packages/ts-etl/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "ts-etl",
+  "version": "0.1.0",
+  "description": "ETL tool for Docmaps",
+  "type": "module",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "ava",
+    "clean": "rm -rf dist/",
+    "test:integration": "ava integration/",
+    "test:unit": "ava src/test/",
+    "start": "node --loader=ts-node/esm --experimental-specifier-resolution=node --nolazy -r ts-node/register/transpile-only src/index.ts",
```

Comment:
I'm so sorry if this took you a long time to find. I can't imagine the countless wasted hours on folks googling why `ts-node` failed with that ESM error.